dockerfile.template

```diff
@@ -28,10 +28,10 @@ RUN pip3 -q install --upgrade pip
 RUN pip3 -q install setuptools
 
 # MSFTKube
-ENV MSFTKUBE_VER 1.0.996349
+ENV MSFTKUBE_VER %%MSFTKUBE_VERSION%%
```

Comment:
I feel like a bit of a dick for suggesting this so late but the other way you could get this is to actually talk to create a release definition (or triggered build) on msdata/vienna that calls a doker build when there's a never green version of msftkube. That way vsts calls you instead of you calling vsts.

Reply:
Yes but this wouldn't retroactively build images for older versions of msftkube if we have changed our dockerfile since.